cleanlab/count.py

```diff
@@ -1098,6 +1095,8 @@ def get_confident_thresholds(labels: np.array, pred_probs: np.array) -> np.array
     labels : np.array
       An array of shape ``(N,)`` of noisy labels, i.e. some labels may be erroneous.
       Elements must be in the set 0, 1, ..., K-1, where K is the number of classes.
+      All unique labels should be present, such that:
```

Comment:
wouldn't "all unique labels": be present by definition?
Maybe reword to: 

"all possible classes in 0, 1,...K-1" must be present in labels"